ksql-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java

```diff
@@ -78,14 +78,15 @@ public void shouldAggregateWithNoWindow() throws Exception {
     final String queryString = String.format(
         "CREATE TABLE %s AS SELECT %s FROM ORDERS WHERE ITEMID = 'ITEM_1' GROUP BY ITEMID;",
         streamName,
-        "ITEMID, COUNT(ITEMID), SUM(ORDERUNITS)"
+        "ITEMID, COUNT(ITEMID), SUM(ORDERUNITS), SUM(KEYVALUEMAP['key2']/2)"
```

Comment:
Consider splitting this off into its own test?

Reply:
These are time intensive integration tests so adding more would increase build time.